packages/idempotency/src/types/IdempotencyOptions.ts

```diff
@@ -111,7 +111,7 @@ type ItempotentFunctionOptions<T extends Array<any>> = T[1] extends Context
  * Options to configure the behavior of the idempotency logic.
  *
  * This is an internal type that is used by the Idempotency utility to
- * configure {@link IdempotencyHandler}.
+ * configure.
```

Comment:
I'd suggest rephrasing this to simply: "This is an internal type that is used for configuration". With the link removed (which I agree with) it's a bit unclear what we are referring to.